test/service/test_manifest.py

```diff
@@ -181,77 +181,6 @@ def _assert_or_create_json_can(self, expected: Path, actual: str):
             with open(expected, 'w') as f:
                 f.write(actual)
 
-
```

Comment:
This hunk looks odd as it indicates that ManifestTestCase was moved, but I think that's a rendering ambiguity.

Pulling up `_assert_pfb` is a good idea but `_assert_tsv` shouldn't be pulled up since it's got nothing to do with cans.